### DISCUSSION TITLE
Plain context menu like react-native-ios-context-menu's ContextMenuButton?

### DISCUSSION THREAD

**Discussion by mozzius** [enesozturk/react-native-hold-menu#102] [Ideas]
Hey! Love this package!

It would be great to have the option to just have the plain context menu without all the blur effects, components moving around etc etc. react-native-ios-context-menu has a ContextMenuButton component for this purpose.

My use case is a menu like this one from twitter:

![image](https://github.com/enesozturk/react-native-hold-menu/assets/10959775/440e7679-cf28-4943-a143-735941118d5b)

I don't want the whole tweet to be tappable, just the "..." icon, but if I do that then everything gets blurred except the icon, which would be weird

**Comment by mozzius** [comment]
This is what it looks like in my app

![Simulator Screenshot - iPhone 14 - 2023-06-06 at 14 56 29](https://github.com/enesozturk/react-native-hold-menu/assets/10959775/fc2777b0-56af-4de4-8b89-6b8973282d06)

**Comment by hadnet** [comment]
Exactly, the blur effect is not always needed, @enesozturk. Please make it optional.
